### PR TITLE
Made website image and meshmate logo visible in readme

### LINKED ISSUE #1095: Meshery.io website screenshot and Meshmates logo in README.md
### Description
 If you take a look at README.md, you'll notice the meshery.io website screenshot missing along with the Meshmates logo

### Expected Behavior
The screenshot and the logo should be visible.

### Screenshots

image

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Adi-ty**
**Description**
This PR makes website image and meshmates logo visible in the readme.md

This PR fixes #1095 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by thisiskaransgit — APPROVED** [review]

**Comment by thisiskaransgit** [comment]
Looks good to merge, thank you for your contribution :tada:

**Review by chinu-anand — CHANGES_REQUESTED** [review]
Please update the screenshot of the meshery.io site with the latest one.

**Comment by Adi-ty** [comment]
> Please update the screenshot of the meshery.io site with the latest one.

Can you provide me with the image to change with the previously existing image of website screenshot

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 <h2>Preview of <a href="https://meshery.io">Meshery.io</a> Website<h2>
 
-![Meshery.io website screenshot](images/mesheryio_screenshot.png)
+![Meshery.io website screenshot](assets/images/mesheryio_screenshot.png)
```
Comment (thisiskaransgit) [inline README.md:31]: > > Please update the screenshot of the meshery.io site with the latest one.
> 
> Can you provide me with the image to change with the previously existing image of website screenshot

@Adi-ty, You shouldn't have asked this question, anyways, I've attached the image, update the screenshot with this existing one.

Screenshot from 2023-04-04 17-45-42





> Please update the screenshot of the meshery.io site with the latest one.

and thank you @chinu-anand, I've overlooked the outdated image

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Adi-ty) [reply README.md:31]: I have changed the image sorry for inconvenience

**Review by chinu-anand — APPROVED** [review]
LGTM, nice work @Adi-ty 👏🏼

**Comment by leecalcote** [comment]
@chinu-anand, thank you.

**Comment by leecalcote** [comment]
@thisiskaransgit, look good?

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

**Comment by Adi-ty** [comment]
Please merge this if everything looks good

**Review by Chadha93 — COMMENTED** [review]
Merge conflicts.

**Comment by vishalvivekm** [comment]
> Merge conflicts.

done by #1109

**Comment by Adi-ty** [comment]
> Merge conflicts.

Resolved merge commits I think now it's good to merge

**Comment by Adi-ty** [comment]
> Merge conflicts.

Resolved merge commits I think now its good to merge

**Comment by leecalcote** [comment]
@thisiskaransgit, how should we proceed with this PR?

**Comment by thisiskaransgit** [comment]
@leecalcote, the visibility change has been taken care of this, change will only update the website preview with the latest screenshot

**Review by thisiskaransgit — APPROVED** [review]

**Comment by Adi-ty** [comment]
Review and merge this PR please.

**Comment by Adi-ty** [comment]
> 18 commits to swap out a file...

All of them are update branch 😂

**Comment by leecalcote** [comment]
oh, I see. Ha! Yes, that makes a difference. 🙂

**Pull request merged**